src/styles/Navbar.module.css

```diff
@@ -0,0 +1,39 @@
+.navContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navTitle {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 2.1rem;
+  margin: 0.5rem 2rem;
+  font-family: "Lato", sans-serif;
+  font-size: 2.6rem;
+  font-weight: bold;
+}
+
+.navlinks {
+  color: #0037ff;
+  font-size: 1.5rem;
+  font-family: "Lato", sans-serif;
+  font-weight: 600;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1rem;
+  padding-right: 2rem;
+}
+
+.navlinks a {
+  color: #037bfe;
```

Comment:
- [ ] @Atifzada great work implementing the changes requested, however, you only missed one requirement, the color of the link as previously highlighted by the previous reviewer, check the image below, the link's color is supposed to match the design given.
![image](https://github.com/Atifzada/Space_Travelers_Hub/assets/61408860/b8c9fa35-4b7d-4bed-8a6a-cd47593712fa)

Reply:
@danmainah Thank you for the review but I already add the color which was asked for. Anyhow I will again look into it.

Reply:
@Atifzada the nav items didn't have te change of color still the same, hope you have implemented the change